app/models/binary/explore/PrecomputedExplorer.scala

```diff
@@ -19,6 +19,7 @@ class PrecomputedExplorer extends RemoteLayerExplorer {
 
   override def explore(remotePath: Path, credentialId: Option[String]): Fox[List[(PrecomputedLayer, Vec3Double)]] =
     for {
+      _ <- Fox.bool2Fox(remotePath.toUri.toString.last == '/') ?~> "Remote dataset explore path must be a directory (indicated by trailing slash)"
```

Comment:
Would it be possible to automatically add a trailing slash in this case so the user does not get this error?

Reply:
If this is specific to the PrecomputedExplorer, I’d say it should happen right here, something like 
`remotePathNormalized <- addTrailingSlashIfMissing(remotePath)` – what do you think?

Reply:
No longer needed since "/" now adds a trailing slash if necessary.